content/docs/command-reference/config.md

```diff
@@ -9,7 +9,7 @@ usage: dvc config [-h] [--global | --system | --local] [-q | -v] [-u]
                   [-l] [name] [value]
 
 positional arguments:
-  name           Option name.
+  name           Option name (section.option or remote.name.option).
```

Comment:
sorry, guys, but you clarify what does `section.option` and `remote.name.option` mean? do we try to refer to the format? should we somehow highlight that this is a format, or may be even better do something like

(e.g. `core.check_update`, `cache.dir`, `remote.myremote.something`)


do we need to specify

Reply:
But if it gets too complicated here maybe we can leave it as it was and explain it in the Options section like for `repro targets` (see #1983)

Reply:
i.e. add a `name` bullet to the top of the Options section and explain it there in detail.

Reply:
Up to you guys, I'm just saying that even for me it took a few moments to parse this and understand the intention.

Reply:
OK I guess it can be clarified in just a couple lines:
```suggestion
  name           Option name in format: section.option or remote.name.option
                 e.g. 'core.check_update', 'cache.dir', 'remote.myremote.url'
```

But now that I read https://dvc.org/doc/command-reference/remote/modify I'm not sure what the change is here, since `remote` was already a config section, so `section.option` kind of covers the `remote.name.option` case. I suppose being explicit is never bad.